test/automated/reflection/variant_actuator/method_is_not_implemented.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../../automated_init'
+
+context "Reflect" do
+  context "Reflection" do
+    context "Variant Actuator" do
+      context "Method Is Not Implemented" do
+        subject = Reflect::Controls::Subject.example
+
+        reflection = Reflect.(subject, :SomeConstant, strict: true)
+
+        random_method_name = Reflect::Controls::Namespace::Random.get
```

Comment:
Same as above re. `:not_implemented_method`.

Also, consider `Controls::MethodName.random` and `Controls::MethodName.example`. But `random_method_name` would not be correct in that case, because just `method_name` would suffice.